packages/web-components/src/components/link-list/link-list-item.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './link-list.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Style schemes
+ */
+export enum STYLE_SCHEME {
+  /**
+   * Default style
+   */
+  DEFAULT = '',
+
+  /**
+   * Link List variant
+   */
+  LINK_LIST = 'link-list',
+}
+
+/**
+ * Link list item.
+ *
+ * @element dds-link-list-item
+ */
+@customElement(`${ddsPrefix}-link-list-item`)
+class DDSLinkListItem extends LitElement {
+  // eslint-disable-next-line class-methods-use-this
+  protected render() {
+    return html`
+      <slot @slotchange=${this._handleSlotChange} />
+    `;
+  }
+
+  connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'listitem');
+    }
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  private _handleSlotChange(e) {
+    e.target?.assignedNodes?.setAttribute('style-scheme', STYLE_SCHEME.LINK_LIST);
```

Comment:
```suggestion
    e.target?.assignedNodes?.forEach(node => { node.styleScheme = STYLE_SCHEME.LINK_LIST; });
```

Please also see my comment to `STYLE_SCHEME` enum.